PinpointKit/PinpointKit/Sources/PinpointKit.swift

```diff
@@ -41,7 +41,8 @@ public class PinpointKit {
      - parameter delegate:           A delegate that is notified of significant events.
      */
     public convenience init(feedbackRecipients: [String], delegate: PinpointKitDelegate? = nil) {
```

Comment:
I don't hate any of those options. The first one has a bit of duplication because we want to use the same defaults as `FeedbackConfiguration`, but don't have any easy way of making the `PinpointKit` initializer defaults the same without duplicating. Maybe a static function for the title that both used would reduce it, but then we could still have a sync issue where the `PinpointKit` init defaults could be out of sync with the `FeedbackConfiguration` defaults as they both change. That might be okay, but it's a scenario I want to raise.